CHANGES.md

```diff
@@ -1,4 +1,4 @@
-## x.x.x
+## 0.7.0
```

Comment:
We haven't done this for previous releases:
https://github.com/BuoyantIO/linkerd/blob/0.6.0/CHANGES.md
https://github.com/BuoyantIO/linkerd/blob/0.5.0/CHANGES.md